### PR TITLE
Disable screensaver for leap 15.x upgrade tw test

### PR DESCRIPTION
**Pull request by rfan1**
- Related ticket: https://progress.opensuse.org/issues/168067
- Needles: n/a
- Verification run: 
[15.0](https://openqa.opensuse.org/tests/4603484)
[15.1](https://openqa.opensuse.org/tests/4603686#step/consoletest_finish/19)

### CODE REVIEW

File: products/opensuse/main.pm [hunk 1]
```diff
@@ -411,6 +411,7 @@ else {
         || is_systemd_test())
     {
         loadtest "console/system_prepare";
+        loadtest "x11/disable_screensaver" if (check_var('FLAVOR', 'NET') && check_var('UPGRADE', '1') && heck_var('ORIGINAL_VERSION', '15.0'));
```
Comment (Vogtinator) [inline products/opensuse/main.pm:414]: Typo

```suggestion
        loadtest "x11/disable_screensaver" if (check_var('FLAVOR', 'NET') && check_var('UPGRADE', '1') && check_var('ORIGINAL_VERSION', '15.0'));
```

Comment (cedvid) [inline products/opensuse/main.pm:414]: Should be `check_var` instead of `heck_var`:
`&& heck_var('ORIGINAL_VERSION'`

Comment (rfan1) [reply products/opensuse/main.pm]: I can find leap 15.1 upgrade test has the same problem. https://openqa.opensuse.org/tests/4601162, let me run VR as well

Comment (rfan1) [reply products/opensuse/main.pm]: Thanks, fixed

Comment (rfan1) [reply products/opensuse/main.pm]: New VR passed as well

**Pull request merged**